packages/fxa-content-server/app/styles/modules/_settings.scss

```diff
@@ -877,6 +877,37 @@ section.modal-panel {
   }
 }
 
+.header-lockup {
+  display: flex;
+  align-items: center;
+}
+
+a.nu-settings-button {
+  background: #FAFAFB;
+  border: 1px solid #C2C2C2;
+  border-radius: .25rem;
+  color: black;
+  height: 100%;
+  margin-left: 1.25rem;
```

Comment:
I think we need a little margin on the right too. maybe `0.5rem`?
<img width="448" alt="Screen Shot 2020-11-03 at 9 26 41 AM" src="https://user-images.githubusercontent.com/87619/98019565-c4539f00-1db6-11eb-845a-c5a312f02c9f.png">